docs/Tutorials/MathComponent/sequences/math_sequence.seq

```diff
@@ -1,9 +0,0 @@
-; Run this sequence to replicate the MathComponent tutorial
```

Comment:
Why was this deleted?  Are we not covering sequences anymore?  Also, it looks out of date.

Reply:
I didn't see anything in the previous version of the tutorial that discussed this sequence. It was just there. I see two options going forward:

1. Revise the sequence to conform to the new tutorial and add a section of the tutorial discussing the sequence.
2. Delete the sequence.

I went with option (2) because sequencing is covered elsewhere and the tutorial already seems long enough. Thoughts?

Reply:
Delete the sequence.  It doesn't even follow the correct naming any more.